frontend/src/screens/wallet/Send.tsx

```diff
@@ -116,156 +135,25 @@ export default function Send() {
         title="Send"
         description="Pay a lightning invoice created by any bitcoin lightning wallet"
       />
-      {hasChannelManagement &&
-        (invoiceDetails?.satoshi || 0) * 1000 >=
-          0.8 * balances.lightning.totalSpendable && (
-          <Alert>
-            <AlertTriangle className="h-4 w-4" />
-            <AlertTitle>Low spending balance</AlertTitle>
-            <AlertDescription>
-              You won't be able to make payments until you{" "}
-              <Link className="underline" to="/channels/outgoing">
-                increase your spending balance.
-              </Link>
-            </AlertDescription>
-          </Alert>
-        )}
+      {hasLowBalance && (
+        <Alert>
+          <AlertTriangle className="h-4 w-4" />
+          <AlertTitle>Low spending balance</AlertTitle>
+          <AlertDescription>
+            You won't be able to make payments until you{" "}
```

Comment:
I think we should remove this alert - we decided to only show the alerts about receiving capacity, as the spending ones are a bit unnecessary. I must have missed this one when removing the other alerts